src/Microsoft.AspNetCore.Mvc.Analyzers.Experimental/ActionsMustNotBeAsyncVoidAnalyzer.cs

```diff
@@ -16,7 +16,7 @@ public class ActionsMustNotBeAsyncVoidAnalyzer : ControllerAnalyzerBase
         public static readonly string ReturnTypeKey = "ReturnType";
 
         public ActionsMustNotBeAsyncVoidAnalyzer()
-            : base(DiagnosticDescriptors.MVC7003_ActionsMustNotBeAsyncVoid)
+            : base(ExperimentalDiagnosticDescriptors.MVC7003_ActionsMustNotBeAsyncVoid)
```

Comment:
Some diamond dependency shennanigans. Renaming the type in the experimental package

Reply:
Where else does `DiagnosticDescriptors` exist? What is it clashing with?

Reply:
https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Analyzers/DiagnosticDescriptors.cs.